### DISCUSSION TITLE
Rustdesk ID server and key integrated in .exe (Windows EXE name)

### DISCUSSION THREAD

**Discussion by Onamorkesem** [rustdesk/rustdesk-server-pro#202] [General]
Hello first of all to all I have read some things on the subject and get stuck I would like to integrate the id server and the key in the app installed by the customer on windows and mac means that the customer should install and do nothing more than start rustdesk unfortunately the codes I have found so far do not help me a bit if someone has already set up the whole thing with himself I ask for a simplified explanation and if possible an example as a photo that I can stick to thank you in advance and a wonderful day to you all :)

**Comment by rustdesk** [comment]
It is already on your web console page. For Windows only, no such thing for mac/linux.

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/71636191/c2be7b4d-efd8-4bef-80cf-748f35d31acb)

**Reply by Onamorkesem** [reply]
we had seen this but we don't quite know what to do with it is this a link or what exactly happens to it

**Reply by rustdesk** [reply]
Download https://github.com/rustdesk/rustdesk/releases/download/1.2.3/rustdesk-1.2.3-x86_64.exe,
rename `rustdesk-1.2.3-x86_64.exe` to `Windows EXE` name you see on your web console.
